packages/lib/src/components/paywall/PayNowForm.svelte

```diff
@@ -114,10 +114,13 @@
     loading = true;
 
     try {
+      const isWallet = ['GOOGLE-PAY', 'APPLE-PAY'].includes(paymentMethod.method || '');
```

Comment:
I find it a bit frustrating that includes doesn't support undefined even though it makes sense. In this case it might be an option to do this first:
`const method = paymentMethod.method ||| 'CARD'`

Then you'll avoid the fallback in two places. It's also a bit scary with passing an empty string as it's not really defined how that would work.